csharp-ovh/Client/Client.cs

```diff
@@ -310,23 +356,7 @@ private WebHeaderCollection GetHeaders(string method, string data, bool needAuth
         /// <exception cref="InvalidResponseException">when API response could not be decoded</exception>
         private string Call(string method, string path, string data = null, bool needAuth = true, bool isBatch = false)
         {
-            PrepareCall(ref method, ref path, data, needAuth, isBatch);
-
-            try
-            {
-                if (method != "GET")
-                {
-                    return _webClient.UploadString(path, method, data ?? "");
-                }
-                else
-                {
-                    return _webClient.DownloadString(path);
-                }
-            }
-            catch (WebException ex)
-            {
-                throw HandleWebException(ex);
-            }
+            return CallAsync(method, path, data, needAuth, isBatch).Result;
```

Comment:
As per [the doc](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.taskawaiter?view=netframework-4.8), these methods are intended to be used by compiler only, so I didn't use them at first. However, I noticed that the move to `.Result` is a breaking change because of the `AggregateException`... So I changed it to `GetResult` anyway to keep the compatibility.